client/src/App.jsx

```diff
@@ -46,38 +47,5 @@ const TransitionRoutes = () => {
 
   console.log(scuntSettings);
   // const {darkMode} = useContext(DarkModeContext);
-  return (
-    <TransitionGroup>
-      {/* <div className={`${darkMode?"dark-mode-body":"light-mode-body"}`}> */}
-      <Navbar isLoggedIn={loggedIn} froshInitials={initials} isRegistered={registered} />
-      <ScrollToTop />
-      <CSSTransition key={location.key} classNames="page" timeout={300}>
-        <Routes location={location}>
-          {[
-            ...pages.main,
-            ...pages.hidden,
-            ...pages.special,
-            ...pages.scunt,
-            ...pages.scuntHidden,
-          ].map((page) => {
-            return (
```

Comment:
uncommit the deletions here
you're welcome to test the component by adding it directly in the App.jsx file, but make sure to remove it after testing -- we'll be setting up another component to display all the landing pages!